diff --git a/dist/node/src/helper.js b/dist/node/src/helper.js
index d90cf42f2575f60f718b8fd0489017d6eeae8938..d9d5d7c6f485f9e5c8dd720d93e514911b987979 100644
--- a/dist/node/src/helper.js
+++ b/dist/node/src/helper.js
@@ -60,7 +60,10 @@ const getEmbedPdfPages = async (arg) => {
     }
     else {
         const willLoadPdf = await (0, common_1.getB64BasePdf)(basePdf);
-        const embedPdf = await pdf_lib_1.PDFDocument.load(willLoadPdf);
+        const embedPdf = await pdf_lib_1.PDFDocument.load(willLoadPdf, {
+            ignoreEncryption: true,
+            password: '',
+        });
         const embedPdfPages = embedPdf.getPages();
         embedPdfBoxes = embedPdfPages.map((p) => ({
             mediaBox: p.getMediaBox(),
